### PR TITLE
remove issue templates

### PR DESCRIPTION
**Pull request by oliver-sanders**
Spotted whilst trawling other folks repos, if you create a repo called ".github" you can configure orginisational defaults for things like issue templates.

> [edit] actually there was an issue for this which I had totally forgotten about:
> https://github.com/cylc/cylc-admin/issues/127

https://github.com/cylc/.github

When these templates are removed from cylc-flow the orginisation defaults will kick in.

See what they look like here: https://github.com/cylc/.github/issues/new/choose

I've used these templates to raise awareness of Discourse by linking through to the site.

(note the report security vulnerability template is GitHub defined)

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` and `conda-environment.yml`.
- [x] Tests are included (or explain why tests are not needed).
- [x] `CHANGES.md` entry included if this is a change that can affect users
- [x] [Cylc-Doc](https://github.com/cylc/cylc-doc) pull request opened if required at cylc/cylc-doc/pull/XXXX.
- [x] If this is a bug fix, PR should be raised against the relevant `?.?.x` branch.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
If approved we can move the pull request template there. It's also possible to configure automations and bots from this repo which might be an interesting possibility going forward.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,28 +1,37 @@
 # Cylc: How to Contribute
 
-## Report Bugs
+Thanks for you interest in the Cylc project!
 
-Report bugs by opening an issue at [Cylc Issues on
-Github](https://github.com/cylc/cylc-flow/issues). Give the Cylc version
-affected by the bug (you should test the latest release if possible) and a
-recipe to reproduce the problem.
```
Comment (oliver-sanders) [inline CONTRIBUTING.md:8]: This advice is now present in the issue templates.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+
+Please read the [CLA](#contributor-licence-agreement-and-certificate-of-origin).
+
+Please add your name to the
+[Code Contributors](#code-contributors) section of this file as part of your
+first Pull Request (for each Cylc repository you contribute to).
 
-Request enhancements by opening an issue at [Cylc Issues @
-Github](https://github.com/cylc/cylc-flow/issues). Describe your use case in
-detail.
```
Comment (oliver-sanders) [inline CONTRIBUTING.md:14]: This advice is now present in the issue templates.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
 
 ## Contribute Code
 
-All contributions to Cylc are made via Pull Requests against the *master*
-branch of [cylc/cylc-flow](https://github.com/cylc/cylc-flow). Non-trivial
```
Comment (oliver-sanders) [inline CONTRIBUTING.md:19]: No longer true.

**Review by hjoliver — APPROVED** [review]
Looks good!

**Comment by oliver-sanders** [comment]
Whoops, made it public: https://github.com/cylc/.github/

**Pull request merged**